package.json

```diff
@@ -19,12 +20,12 @@
   "private": false,
   "scripts": {
     "test": "./scripts/run-and-test.sh",
-    "only-run-tests": "ts-node node_modules/jasmine/bin/jasmine --config=jasmine.json",
-    "buf": "./buf.gen.yaml",
+    "only-run-tests": "vitest",
+    "buf": "buf generate && tsc-esm-fix --src src/authzedapi --ext='.js'",
```

Comment:
the tsc-esm-fix is needed until https://github.com/timostamm/protobuf-ts/issues/656 is fixed.